From ad4ca6cd5be79a8c7c10b3648fa4ff4fb082e63b Mon Sep 17 00:00:00 2001
From: Boris Savelev <boris@altlinux.org>
Date: Wed, 28 Mar 2012 12:41:34 +0400
Subject: [PATCH 01/23] 0.16-alt17.svn708
To: wine-patches <wine-patches@winehq.org>
Reply-To: wine-devel <wine-devel@winehq.org>

- update from trunk
---
 opennx.spec | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/opennx.spec b/opennx.spec
index 817de71..0519fbc 100644
--- a/opennx.spec
+++ b/opennx.spec
@@ -4,7 +4,7 @@
 Summary: An OpenSource NX client
 Name: opennx
 Version: 0.16
-Release: alt16.svn634
+Release: alt17.svn708
 License: LGPL/GPL
 Group: Networking/Remote access
 Url: http://sourceforge.net/projects/opennx
@@ -22,6 +22,7 @@ Requires: usbip2-nxclient
 BuildRequires: gcc-c++ imake libSM-devel libXmu-devel nx ImageMagick-tools
 BuildRequires: libopensc-devel libsmbclient-devel
 BuildRequires: libwxGTK-devel xorg-cf-files zip libcups-devel
+BuildRequires: libXau-devel
 
 %description
 opennx is an OSS replacement for Nomachine's NX client.
@@ -122,6 +123,9 @@ install -m 644 etc/*.rules %buildroot%_sysconfdir/udev/rules.d
 %endif
 
 %changelog
+* Wed Mar 28 2012 Boris Savelev <boris@altlinux.org> 0.16-alt17.svn708
+- update from trunk
+
 * Fri Jun 17 2011 Boris Savelev <boris@altlinux.org> 0.16-alt16.svn634
 - update from trunk
 
-- 
2.1.4

